WORKSPACE

```diff
@@ -12,36 +12,6 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
-load("//build:copts.bzl", "EXTRA_COPTS")
-
-http_archive(
-    name = "aspect_bazel_lib",
-    sha256 = "a7e356f8a5cb8bf1e9be38c2c617ad22f5a1606792e839fc040971bdfbecf971",
-    strip_prefix = "bazel-lib-1.40.2",
-    url = "https://github.com/aspect-build/bazel-lib/archive/refs/tags/v1.40.2.tar.gz",
-)
-
-http_archive(
-    name = "gcc_toolchain",
-    sha256 = "e6a00a9f999b29ba4eee0bf73f74abeeec1cb85740c8c197a8bfa89a73e722b0",
-    strip_prefix = "gcc-toolchain-0.9.0",
-    urls = [
-        "https://github.com/f0rmiga/gcc-toolchain/releases/download/0.9.0/gcc-toolchain-0.9.0.tar.gz",
-    ],
-)
-
-load("@gcc_toolchain//toolchain:defs.bzl", "ARCHS", "gcc_register_toolchain")
-
-gcc_register_toolchain(
-    name = "gcc_toolchain_x86_64",
-    extra_cflags = EXTRA_COPTS,
-    extra_ldflags = ["-l:libstdc++.a"],
-    gcc_version = "14.3.0",
-    target_arch = ARCHS.x86_64,
-    target_compatible_with = ["@//build/compiler:gcc_14"],
```

Comment:
I'm guessing this was the old way that we said "`--config=gcc14` uses this toolchain".